backend/src/backend/routes/books.py

```diff
@@ -110,6 +110,13 @@ async def create_book(book_payload: BookAddSchema):
         await book.languages.add(language)
         await title.languages.add(language)
 
+    await clean_old_books(
```

Comment:
I think this should be wrapped in a try block as this is a subsidiary, internal operation that we trigger upon new book creation. We wouldn't want the book addition to fail (and thus everything rolled-back) just because the obsolete book deletion failed. This should just trigger a log entry.